### ISSUE TITLE
[BUG] System.IO.IOException: Access to the path '\\?\C:\Users\User\AppData\Local\Bloxstrap\Roblox\Player.old\content\textures\Cursors\KeyboardMouse' is denied.

### ISSUE THREAD

**Issue by DesignerTea** [bloxstraplabs/bloxstrap#3592]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.
- [X] I have thoroughly looked through the available Wiki articles and could not find a solution to my problem.
- [X] I am using the latest version of Bloxstrap.
- [ ] I did not answer truthfully to all the above checkboxes.

### Bloxstrap Version

v2.8.1

### What problem did you encounter?

![image](https://github.com/user-attachments/assets/4f7cf3ff-a980-4e3d-975a-e7179d08cf30)
hey there
i am still have a problem with my bloxstrap when i try to join a game

### Bloxstrap Log

```text
2024-10-31T12:27:05Z [App::OnStartup] Starting Bloxstrap v2.8.1
2024-10-31T12:27:05Z [App::OnStartup] Compiled Wednesday, 30 October 2024 at 1:35:23 AM from commit 2be2f4d0d275e60eabf2f63ecbbe4e38b2aa0773 (tag/v2.8.1)
2024-10-31T12:27:05Z [App::OnStartup] Loaded from %UserProfile%\AppData\Local\Bloxstrap\Bloxstrap.exe
2024-10-31T12:27:05Z [App::OnStartup] Temp path is %UserProfile%\AppData\Local\Temp\Bloxstrap
2024-10-31T12:27:05Z [App::OnStartup] WindowsStartMenu path is %UserProfile%\AppData\Roaming\Microsoft\Windows\Start Menu\Programs
2024-10-31T12:27:05Z [Logger::Initialize] Initializing at %UserProfile%\AppData\Local\Bloxstrap\Logs\Bloxstrap_20241031T122705Z.log
2024-10-31T12:27:05Z [Logger::Initialize] Finished initializing!
2024-10-31T12:27:05Z [JsonManager<Settings>::Load] Loading from %UserProfile%\AppData\Local\Bloxstrap\Settings.json...
2024-10-31T12:27:05Z [JsonManager<Settings>::Load] Loaded successfully!
2024-10-31T12:27:05Z [JsonManager<State>::Load] Loading from %UserProfile%\AppData\Local\Bloxstrap\State.json...
2024-10-31T12:27:05Z [JsonManager<State>::Load] Loaded successfully!
2024-10-31T12:27:05Z [FastFlagManager::Load] Loading from %UserProfile%\AppData\Local\Bloxstrap\Modifications\ClientSettings\ClientAppSettings.json...
2024-10-31T12:27:05Z [FastFlagManager::Load] Loaded successfully!
2024-10-31T12:27:06Z [LaunchHandler::LaunchRoblox] Initializing bootstrapper
2024-10-31T12:27:06Z [LaunchHandler::LaunchRoblox] Initializing bootstrapper dialog
2024-10-31T12:27:06Z [Bootstrapper::Run] Running bootstrapper
2024-10-31T12:27:06Z [Bootstrapper::SetStatus] Connecting to Roblox...
2024-10-31T12:27:06Z [Deployment::TestConnection<https://setup.rbxcdn.com>] Connecting...
2024-10-31T12:27:06Z [HttpClientLoggingHandler::ProcessRequest] GET https://setup.rbxcdn.com/versionStudio
2024-10-31T12:27:06Z [Deployment::InitializeConnectivity] Testing connectivity...
2024-10-31T12:27:07Z [HttpClientLoggingHandler::ProcessResponse] 200 OK https://setup.rbxcdn.com/versionStudio
2024-10-31T12:27:07Z [Deployment::InitializeConnectivity] Got https://setup.rbxcdn.com as the optimal base URL
2024-10-31T12:27:07Z [Bootstrapper::Run] Connectivity check finished
2024-10-31T12:27:07Z [Bootstrapper::CheckForUpdates] Checking for updates...
2024-10-31T12:27:07Z [HttpClientLoggingHandler::ProcessRequest] GET https://api.github.com/repos/bloxstraplabs/bloxstrap/releases/latest
2024-10-31T12:27:07Z [HttpClientLoggingHandler::ProcessResponse] 200 OK https://api.github.com/repos/bloxstraplabs/bloxstrap/releases/latest
2024-10-31T12:27:07Z [Bootstrapper::CheckForUpdates] No updates found
2024-10-31T12:27:07Z [Bootstrapper::GetLatestVersionInfo] Got channel as production
2024-10-31T12:27:07Z [Deployment::GetInfo] Getting deploy info for channel production
2024-10-31T12:27:07Z [HttpClientLoggingHandler::ProcessRequest] GET https://clientsettingscdn.roblox.com/v2/client-version/WindowsPlayer
2024-10-31T12:27:08Z [HttpClientLoggingHandler::ProcessResponse] 200 OK https://clientsettingscdn.roblox.com/v2/client-version/WindowsPlayer
2024-10-31T12:27:08Z [RegistryKeyEx::SetValueSafe] Writing '' to HKEY_CURRENT_USER\SOFTWARE\ROBLOX Corporation\Environments\RobloxPlayer\Channel\www.roblox.com
2024-10-31T12:27:08Z [HttpClientLoggingHandler::ProcessRequest] GET https://setup.rbxcdn.com/version-b7eebc919e96477a-rbxPkgManifest.txt
2024-10-31T12:27:08Z [HttpClientLoggingHandler::ProcessResponse] 200 OK https://setup.rbxcdn.com/version-b7eebc919e96477a-rbxPkgManifest.txt
2024-10-31T12:27:08Z [Bootstrapper::SetStatus] Upgrading {product}...
2024-10-31T12:27:08Z [LaunchHandler::LaunchRoblox] Bootstrapper task has finished
2024-10-31T12:27:08Z [LaunchHandler::LaunchRoblox] An exception occurred when running the bootstrapper
2024-10-31T12:27:08Z [App::FinalizeExceptionHandling] (0x80131620) System.IO.IOException: Access to the path '\\?\%UserProfile%\AppData\Local\Bloxstrap\Roblox\Player.old\content\textures\Cursors\KeyboardMouse' is denied.
   at System.IO.FileSystem.RemoveDirectoryRecursive(String fullPath, WIN32_FIND_DATA& findData, Boolean topLevel)
   at System.IO.FileSystem.RemoveDirectory(String fullPath, Boolean recursive)
   at Bloxstrap.Bootstrapper.UpgradeRoblox()
   at Bloxstrap.Bootstrapper.Run()
   at Bloxstrap.Bootstrapper.Run()
2024-10-31T12:27:08Z [HttpClientLoggingHandler::ProcessRequest] POST https://bloxstraplabs.com/metrics/post-exception
2024-10-31T12:27:08Z [HttpClientLoggingHandler::ProcessResponse] 429 Too Many Requests https://bloxstraplabs.com/metrics/post-exception
```

**Comment by bluepilledgreat** [comment]
Could you go to `Computer\HKEY_CURRENT_USER\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\Bloxstrap` in a registry editor and show the value of `InstallLocation`?

**Comment by bluepilledgreat** [comment]
I'm assuming this has been fixed with v2.8.2. If not, please open a new issue.
